pact/relay/pool.pact

```diff
@@ -90,6 +104,17 @@
   (defcap ACTIVITY ( pool:string bond:string activity:integer)
     @event true)
 
+
+  (defcap ROTATE (bond:string account:string)
```

Comment:
We can take out `account` from the parameter. Or, is it intended to so people know what they're signing for?

Reply:
Yeah, it's doubling as an event

Reply:
Why don't we use different name for `account` in  line 112 and line 115? Something like orig-acct ? Just to be clear about which guard we're enforcing.